tests/all

```diff
@@ -6,7 +6,22 @@ root=$tdir/..
 
 cd -- "$root"
 
-for t in tests/test-*.scm; do
-	printf '* %s:\n' "$t"
-	csi -s "$t"
+# List of schemes to use.  For each scheme you need to provide a intprt_SCHEME
```

Comment:
nit: For a public function like this that is actually referenced rarely, maybe avoid the abbrev and just use `interpret_SCHEME`?